server/services/omnichannel-voip/service.ts

```diff
@@ -17,12 +23,71 @@ export class OmnichannelVoipService extends ServiceClass implements IOmnichannel
 
 	private users: UsersRaw;
 
+	private voipRoom: VoipRoomsRaw;
+
 	constructor(db: Db) {
 		super();
 		this.users = new UsersRaw(db.collection('users'));
+		this.voipRoom = new VoipRoomsRaw(db.collection('rocketchat_room'));
 		this.logger = new Logger('OmnichannelVoipService');
 	}
 
+	private normalizeAgent(agentId: string): Record<string, unknown> | undefined {
```

Comment:
The return value is not unknown. It's an object with `_id`, `name`, `username`, `phone`, `status`, `livechat` and `emails?` attributes.

Reply:
Fixed.